src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -870,4 +870,7 @@
   <data name="ColorPicker_ColorFormatsDescription.Text" xml:space="preserve">
     <value>Editor color formats (Change the order by dragging)</value>
   </data>
+  <data name="ColorPicker_ShowColorName.Content" xml:space="preserve">
+    <value>Show color name</value>

```

Comment:
This describes the setting better:
```suggestion
    <value>Show color name in the picker window</value>
```


@crutkas What ist the correct phrase: "... in the picker window" or "... on the picker window"?

Reply:
Not sure if this is needed, we also don't say anything about picker window in color format combobox

Reply:
But in combobox we say something about clipboard.